main.py

```diff
@@ -13,93 +12,45 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# [START app]
+import datetime
 import logging
-import psycopg2
+import os
+import json
 
 from flask import Flask
-from flask import jsonify
-from flask_cors import CORS
-
-# DB Connection
-conn = psycopg2.connect("host=34.97.189.196 dbname=photo user=root password=yori1234")
-cur = conn.cursor()
-
-  
-# test data
-dummy0 = {
-    "photo_group": "乃木坂46",
-    "photo_member": "樋口日奈",
-    "photo_costume": "2019 summer concert 1",
-    "photo_type": "ヨリ",
-    "photo_number": 1,
-    "photo_folder": "summer concert",
-    "photo_tag": []
-}
-
-dummy1 = {
-    "photo_group": "乃木坂46",
-    "photo_member": "樋口日奈",
-    "photo_costume": "2019 summer concert 1",
-    "photo_type": "チュウ",
-    "photo_number": 3,
-    "photo_folder": "summer concert",
-    "photo_tag": []
-}
-
-dummy2 = {
-    "photo_group": "乃木坂46",
-    "photo_member": "樋口日奈",
-    "photo_costume": "2019 summer concert 1",
-    "photo_type": "ヒキ",
-    "photo_number": 1,
-    "photo_folder": "summer concert",
-    "photo_tag": []
-}
-
-photos = [dummy0, dummy1, dummy2]
+import sqlalchemy
 
 app = Flask(__name__)
-CORS(app)
-app.config["JSON_AS_ASCII"] = False
-
-@app.route('/')
-def hello():
-    """Return a friendly HTTP greeting."""
-    return 'Hello World!'
 
+logger = logging.getLogger()
 
-@app.route('/photos/all', methods=['GET'])
-def photos_all():
-    return jsonify(photos)
+db_user = os.environ.get("DB_USER")
+db_pass = os.environ.get("DB_PASS")
+db_name = os.environ.get("DB_NAME")
+cloud_sql_connection_name = os.environ.get("CLOUD_SQL_CONNECTION_NAME")
 
-
-@app.route('/create/photo', methods=['POST', 'GET'])
-def create_photo(): #生写真の追加
-    if request.method == 'POST': 
-        cur.execute("SELECT COUNT(id) FROM photo")
-        rows = cur.fetchall()
-        for row in rows:
-            ids = rows[0] + 1
-        cur.execute("INSERT INTO photos (photo_group, photo_member, photo_costume, photo_type, photo_number, photo_folder, photo_tag) VALUES (%s, %s, %s, %s, %d, %s, %s)", request.form['group'], request.form['member'], request.form['costume'], request.form['type'], request.form['number'], request.form['folder'], request.form['tag'])
-        cur.commit()
-        return redirect(url_for('display'))
+def connect_to_cloudsql():
+    # When deployed to App Engine, the `SERVER_SOFTWARE` environment variable
+    # will be set to 'Google App Engine/version'.
+    if os.getenv('SERVER_SOFTWARE', '').startswith('Google App Engine/'):
```

Comment:
should be 
```python
    if os.getenv('GAE_ENV') == 'standard':
```

Reply:
thanks
https://yori-server.appspot.com/env